app/models/course.rb

```diff
@@ -6,4 +6,13 @@ class Course < ActiveRecord::Base
   belongs_to :creator, class_name: User.name
   has_many :course_users, inverse_of: :course, dependent: :destroy
   has_many :users, through: :course_users
+
+  before_validation :set_default_values, on: :create
```

Comment:
I think this should be an `after_initialize` callback, without an `on` clause. Because when I have a `new Course`, I want the defaults already set. I do not want to wait until `save` is called for my defaults to appear, right?